jupytercalpoly/Jupyter-Bifrost
# Range Slider

**Adds a better UI component to deal with filtering along quantitative ranges**
## Changes
- [x] Created range slider
- [x] Updated Quantitative Filters to use range slider
- [x] Added interactive value indicators to each side of the slider.
- [x] Style the element
- [x] Add multiple ranges to Quantitative Filters
- [x] Add and delete ranges in Quantitative Filters
- [x] Update "Export Code" to handle compound ranges
- [x] Abstracted `ModelStateName` type to derive its type directly from the `BifrostModel` class.

## Testing
1. Open filters on a quantitative encoded variable.
2. Change the min and max by pulling the handles on the slider
3. Change the min and max by changing the inputs to each side of the slider and hitting Enter
4. Hover over the slider track and see the value being displayed
5. Click on the track and verify that the nearest bound jumps to the position.
6. Add a filter and ensure that the final range is the combination of the ranges of the two filters.
7. Delete both filters and add once back. Ensure that it still works
8. Export code and ensure that the result runs correctly.
